dumper/lib-dumper-spi/src/main/java/com/google/edwmigration/dumper/plugin/lib/dumper/spi/HdfsPermissionExtractionDumpFormat.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2022-2024 Google LLC
+ * Copyright 2013-2021 CompilerWorks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.edwmigration.dumper.plugin.lib.dumper.spi;
+
+public interface HdfsPermissionExtractionDumpFormat {
+  String FORMAT_NAME = "hdfs-permissions";
```

Comment:
I would recommend changing this value to `hdfs-permissions.dump.zip`, so that it differs from the connector name and that it is similar to existing format names from other `*DumpFormat` classes.